_posts/2020-03-15-learning-to-share-accomplishments-2.md

```diff
@@ -0,0 +1,214 @@
+---
+title: "Learning to Share Accomplishments: Part 2 - Sharing My Accomplishments"
+date: 2020-03-15T10:00:00-04:00
+permalink: /2020/03/15/learning-to-share-accomplishments-2/
+author_profile: true
+toc: true
+toc_label: "Post Contents"
+toc_icon: "file-alt"
+medium_post:
+  - ""
+categories:
+  - Accomplishments
+  - Job Hunting
+tags:
+  - 
+---
+
+This post is a part of a series on job hunting and job interviewing.
+{: .notice--info}
+
+<figure>
+    <img src='https://www.hdwallpaper.nu/wp-content/uploads/2017/04/cat-11.jpg' alt='Until I figure out a better image' width="50%" height="50%" />
+    <figcaption>Until I figure out a better image to put here.....</figcaption>
+</figure>
+This is the second part of a two-part post. The first part is more on how to share your career accomplishments. You 
+may want to read it first. [Part 1 is here.](../learning-to-share-accomplishments-1/)
+
+For part 2 of this post, I want to actually talk about those accomplishments. I think it's helpful for me to get out 
+there and while it may not be as interesting or relevant for you, maybe in looking it over you can see some ways you 
+can talk about your own accomplishments as well. 
+
+## What I Personally Have Worked on in My Past Jobs
+
+I want to spend some time looking back at my work. Like I said above, I've apparently had some times I've been terrible 
+at talking about this in job interviews, so I wanted a list. I kind of want to use it as a thought exercise for me, 
+but maybe also to help sell myself to anyone who might be reading this post. I don't intend this to be literally 
+everything I've worked on, but perhaps more of the highlights from different points in my career. And who knows, maybe 
+a future employer might see this and think I really would be an awesome addition to their team.
+
+One other thing: I have literally had job interviews where, despite all of my developer experience, have looked at me 
+and said "Looks like you do a lot of teaching stuff. Maybe you should be a teacher and not a developer." I've found 
```

Comment:
Oh sweet Jesus. Why wouldn't you want a skilled dev who loves to teach? THINK OF THE DOCS THEY WILL WRITE.
I bet this comes back to the story people love to believe of the "passionate" dev who doesn't see anything outside of their terminal. Ugh.

Reply:
I think some of it is gender related. Because I can't imagine guys get this very often.